src/handler.ts

```diff
@@ -20,18 +22,12 @@ const handler = async (event: DynamoDBStreamEvent, _context: Context, callback:
     const secrets: string[] = await getSecret(process.env.SECRET_NAME);
 
     event.Records.forEach((record) => {
-      if (secrets.includes(record.dynamodb.testStationPNumber as string)) {
-        if (record.dynamodb.testTypeEndTimestamp !== '') {
-          // Send test result
-          const testActivity: TestActivity = dataFormatter(record);
-          logger.info(testActivity);
-        } else {
-          logger.info(
-            `Test result associated with EventID: ${
-              record.eventID
-            } was not completed, event has not been sent to EventBridge`,
-          );
-        }
+      if (secrets.includes(getTestStationNumber(record))) {
+        const testActivity: TestActivity[] = formatDynamoData(record);
+        logger.info(testActivity);
```

Comment:
Not sure we want to log every record?

Reply:
Agreed, another missed testing one, apologies